fury/data/fetcher.py

```diff
@@ -370,6 +379,48 @@ def fetcher():
     )
 
 
+def read_viz_cubemap(name, suffix_type=1, ext='.jpg'):
+    """Read specific cube map with specific suffix type and extension.
+
+    Parameters
+    ----------
+    name : str
+    suffix_type : int, optional
+        0 for numeric suffix (e.g., skybox_0.jpg, skybox_1.jpg, etc.), 1 for
+        -p/nC encoding where C is either x, y or z (e.g., skybox-px.jpeg,
+        skybox-ny.jpeg, etc.), 2 for pos/negC where C is either x, y, z (e.g.,
+        skybox_posx.png, skybox_negy.png, etc.), and 3 for position in the cube
+        map (e.g., skybox_right.jpg, skybox_front.jpg, etc).
+    ext : str, optional
+        Image type extension. (.jpg, .jpeg, .png, etc.).
+
+    Returns
+    -------
+    list of paths : list
+        List with the complete paths of the skybox textures.
+
+    """
+    # Set of commonly used cube map naming conventions and its associated
+    # indexing number. For a correct creation and display of the skybox,
+    # textures must be read in this order.
+    suffix_types = {
+        0: ['0', '1', '2', '3', '4', '5'],
+        1: ['-px', '-nx', '-py', '-ny', '-pz', '-nz'],
+        2: ['posx', 'negx', 'posy', 'negy', 'posz', 'negz'],
+        3: ['right', 'left', 'top', 'bottom', 'front', 'back']
+    }
+    if suffix_type in suffix_types:
+        conv = suffix_types[suffix_type]
+    else:
+        warnings.warn('read_viz_cubemap(): Invalid suffix_type.')
+        return None
```

Comment:
you should select one by default and continue the process

Reply:
Done.